WISE-Community/WISE-Client
# feat(Table): Allow authors to specify frozen columns

## Changes
Adds a checkbox for each column in Table component authoring that specifies whether the column in frozen or not.

Tabulator allows any number of frozen columns on each side of the table. Frozen columns must be next to each other. Once a non-frozen column is reached, Tabulator stops building the frozen list for that side of the table. Total number of frozen columns must be less than total number of columns.

Closes #715.

## Test
- When authoring a Table component Select the "Freeze" checkbox for any columns in the table.
- Verify in VLE (and grading) that any columns you checked are frozen in Tabulator (if they're next to each other).
- Verify that you can't select all of the columns to be frozen. (If you select n-1 frozen columns, the last remaining column should have it's 'Frozen' checkbox disabled.)
